location/models.py

```diff
@@ -44,16 +44,19 @@ def representative_image_or_placeholder(self):
         image = self.representative_image
         return image if image is not None else "/static/img/no_img.png"
 
-    @property
-    def rent_price_for_display(self):
+    def number_of_bed_for_display(self):
```

Comment:
Is there a reason why you're removing the `rent_price_for_display` method and replacing it with `number_of_bed_for_display`? I imagine that method is still useful to other parts of the code base.